code/provision/playbooks/roles/rke-kubernetes/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+cluster_name: cehsos-k8s
+
+rke_kubernetes_version: v1.15.6-rancher1-2
+
+rke_release: v1.0.2
```

Comment:
Probably worth using the latest versions of Kubernetes (1.17) & RKE if they're available? DataLabs should work fine on the latest versions now I believe

Reply:
Would bump up to 1.17 for ceh-datalab tenancy